packages/client/src/intros/shared/Intro.tsx

```diff
@@ -76,25 +76,24 @@ export default ({
       {exceptions.length > 0 && (
         <>
           <Heading styleAs="h4" forwardedAs="h3">
-            {t("introPage.common.exceptions title")}
+            <em>{t("introPage.common.exceptions title")}</em>
           </Heading>
-          <Paragraph gutterBottom={8}>
-            {t("introPage.common.exceptions description")}
-          </Paragraph>
+          {showContactInformation && (
+            <Paragraph>
```

Comment:
This Paragraph should have a smaller gutter bottom:

---
![image](https://user-images.githubusercontent.com/7781865/110338286-1e939380-8027-11eb-8ad7-dad128c65e38.png)